model_compression_toolkit/core/pytorch/utils.py

```diff
@@ -17,6 +17,8 @@
 from typing import Union
 from model_compression_toolkit.core.pytorch.constants import CUDA, CPU
 
+_device = CUDA if torch.cuda.is_available() else CPU
```

Comment:
These changes are not relevant to this PR. 
Please remove the changes of utils.py from this changeset.

Reply:
Agree it's not relevant to this changeset but it's needed and pending on my other PR.
So I suppose when the other PR is merged, this change disappears....

Reply:
The other PR need some fixes, please see the comments I left you there.
I just think that there is no reason that this PR would need to wait because the other one is on hold, so you can consider removing these changes from here so we can merge it.